qase-robotframework/src/qaseio/robotframework/listener.py

```diff
@@ -29,17 +28,18 @@ def __init__(self):
 
             if not token or not project_code:
                 raise ValueError("Token and Project code should be provided")
-            
+
             run_title = "Automated Run {}".format(str(datetime.now()))
             if (self._get_param(Envs.TESTOPS_RUN_TITLE, None)):
                 run_title = self._get_param(Envs.TESTOPS_RUN_TITLE, None)
-            
+
             self.reporter = QaseTestOps(
                 api_token=token,
                 project_code=project_code,
                 run_id=self._get_param(Envs.TESTOPS_RUN_ID, None),
                 plan_id=self._get_param(Envs.TESTOPS_PLAN_ID, None),
-                complete_run=self._get_param(Envs.TESTOPS_COMPLETE_RUN) and self._get_param(Envs.TESTOPS_COMPLETE_RUN).lower() in ['true', '1'],
+                complete_run=self._get_param(Envs.TESTOPS_COMPLETE_RUN) and self._get_param(
```

Comment:
could be 
```
self._get_param(Envs.TESTOPS_COMPLETE_RUN, false).lower() in ['true', '1']
```

but I don't insist )

Reply:
Two things happen here:

1. We check if the variable is set.
2. If it's set, we check if it's true.

It's the same as if we use `_get_param` with `false) as default value.

Reply:
Overall, it seems like a very complicated way to check if config variable is `True`. All these conversions should happen when we read the config, not when we use it.